proxy/src/control/destination/mod.rs

```diff
@@ -192,6 +243,16 @@ where
     }
 }
 
+// ===== impl Responder =====
+
+impl Responder {
+    fn is_active(&self) -> bool {
+        let weak = Arc::weak_count(&self.active);
+        debug_assert!(weak == 0 || weak == 1);
```

Comment:
TIOLI:
```rust
debug_assert!(weak <= 1);` might be clearer, we know `weak_count` won't be < 0 because it's unsigned...
```

Reply:
that's less clear to me ;) as written here, i know it's exactly one of those two states. with `<= 1`, i need to think about types, etc to understand the possible states.

Reply:
fair! :)